### DISCUSSION TITLE
Please please please change the main red color to something else.

### DISCUSSION THREAD

**Discussion by austinmpask** [TeamNewPipe/NewPipe#10966] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

The app is using the "muted" color scheme which was popular in 2014 android apps. Please make the main app color to something more punchy to conform to more modern design conventions. It looks like puke.

**Comment by mcmurphy8097234789** [comment]
It would be more reasonable to request (or create a pull request) for different color themes/color theme manager (so everyone could set the color and theme they prefer, which would be a useful feature), rather than just saying 'i DoN't LiKe ThE cOlOrS pLs NeRf"

**Comment by Pfeil** [comment]
I'd propose to use the android accent color, if available (on my phone it takes this color from my wallpaper and uses it in system ui and many apps), and derive theme colors from it. If there is no accent color, it would use a default one. The color could also be easily customized.

**Comment by opusforlife2** [comment]
> It looks like puke.

It has always struck me as strange why people have such strong opinions about these things at all. I don't care what colours my various apps have. The most I'm likely to ask for is a dark theme to reduce eye strain.
